lightwood/api/json_ai.py

```diff
@@ -979,8 +979,8 @@ def code_from_json_ai(json_ai: JsonAI) -> str:
 # Extract data
 # --------------- #
 # Extract the featurized data
-encoded_old_data = new_data['old']
```

Comment:
the reason the signature here was this way was because the output of the splitter is a dict